packages/contracts/contracts/utils/UUPSProxy.sol

```diff
@@ -3,15 +3,19 @@
 pragma solidity 0.8.10;
 
 import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import "./AppStorage.sol";
 
 /// @title UUPSProxy
-/// @author Aragon Association - 2022
+/// @author Giorgi Lagidze - Aragon Association - 2022
```

Comment:
Aren't contributions recorded on Git? If the rest of files are authored by the association, I suggest we leave this one as is

Reply:
done. we used to do it previously and I kept doing it again.